Riot/Modules/Common/Recents/Views/RecentTableViewCell.m

```diff
@@ -92,12 +92,17 @@ - (void)render:(MXKCellData *)cellData
         self.lastEventDecriptionLabelTrailingConstraint.constant = self.unsentImageView.hidden ? 10 : 30;
 
         // Notify unreads and bing
-        if (roomCellData.hasUnread)
+        if (roomCellData.isRoomMarkedAsUnread)
```

Comment:
I can't say I'm particularly happy with how convoluted the logic in this file is but let's leave it alone, just complaining 🙃

Reply:
yes, probably is only bad naming... 
I should expose something like isGreenDotVisible?

Reply:
Neah, leave it alone, it's not worth the trouble.